sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -562,18 +560,19 @@ public List<EventParticipantIndexDto> getIndexList(
 				case EventParticipantIndexDto.LAST_NAME:
 				case SampleIndexDto.PATHOGEN_TEST_RESULT:
 					expression = joins.getSamples().get(SampleIndexDto.PATHOGEN_TEST_RESULT);
-					order.add(sortProperty.ascending ? cb.asc(expression) : cb.desc(expression));
 					break;
 				case SampleIndexDto.SAMPLE_DATE_TIME:
 					expression = joins.getSamples().get(SampleIndexDto.SAMPLE_DATE_TIME);
-					order.add(sortProperty.ascending ? cb.asc(expression) : cb.desc(expression));
 					break;
 				case EventParticipantIndexDto.FIRST_NAME:
 					expression = person.get(sortProperty.propertyName);
 					break;
 				case EventParticipantIndexDto.CASE_UUID:
 					expression = resultingCase.get(Case.UUID);
 					break;
+				case EventParticipantIndexDto.CONTACT_COUNT:
```

Comment:
This a) doesn't make sense to me (because you're adding sorting to a property, but then sort by something entirely different), and b) doesn't have any effect because the column is set as not sortable anyway. Should be removed.

Reply:
was left there by mistake, saw the column is not sortable after and forgot to remove the code